discord/enums.py

```diff
@@ -218,6 +218,11 @@ def __str__(self) -> str:
         return self.name
 
 
+class MessageReferenceType(Enum):
+    default = 0
```

Comment:
Is `default` the name for a reply reference in the API, or is this an <us> creation?
If the latter, I believe it can be named better, like `reply`.

Reply:
Default is the name that the Discord API itself uses to call them